cssom-view/Overview.bs

```diff
@@ -1352,11 +1352,14 @@ The <dfn attribute for=HTMLElement>offsetParent</dfn> attribute must return the
     * The element is the root element.
     * The element is <a>the HTML <code>body</code> element</a>.
     * The element's computed value of the 'position' property is ''position/fixed''.
-1. Return the nearest ancestor <a lt="unclosed node">unclosed</a> element of the element for which at least one of the following is true and terminate this algorithm if such an ancestor is found: [[!DOM]]
-    * The computed value of the 'position' property is not ''static''.
-    * It is <a>the HTML <code>body</code> element</a>.
-    * The computed value of the 'position' property of the element is ''static'' and the ancestor is one of the following <a>HTML elements</a>: <code>td</code>, <code>th</code>, or <code>table</code>.
-1. Return null.
+1. Let <var>ancestor</var> be the parent of the element and repeat these substeps:
+    1. If <var>ancestor</var> is not an <a lt="unclosed node">unclosed</a> element of the element and its computed value of the 'position' property is ''position/fixed'', terminate this algorithm and return null.
```

Comment:
"element of the element"? I assume this is copy/paste leftover from the previous text, and should delete the "of the element" part.